### PR TITLE
autopkg: add page

### PR DESCRIPTION
**Pull request by Nerthil**
initial

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by fuerbringer** [comment]
tldr-pages are written in the `markdown` format and thus should have the file ending `.md`. Your page also has a couple formatting mistakes (which should've been picked up by CI, but tldr-lint has not [implemented asserting .md yet](https://github.com/tldr-pages/tldr-lint#todo))

@Nerthil I recommend [cloning your fork](https://help.github.com/articles/cloning-a-repository/) and checking out your branch locally and use the following tools to get an idea how the page will end up looking:

- `tldr-lint ./pages/osx/autopkg` will give you formatting hints ([preview](https://gist.github.com/fuerbringer/91b215b86209469194b536710f5c90de))
- `tldr --render ./pages/osx/autopkg` to render the page and see how it will end up looking ([preview](https://gist.github.com/fuerbringer/a329ff10f76e9832e463842657e00481))

File: pages/osx/autopkg [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+`autopkg run {{recipe_identifier}} --pkg {{/path/to/downloaded/pkg_or_dmg}}`
+
+- Update one or more recipe repo:
+
+`autopkg repo-update {{repo_names}}`
+
+`autopkg repo-update all`
+
+- Search avaible recipes:
```
Comment (fuerbringer) [inline pages/osx/autopkg]: Typo, I guess you meant `available`.

File: pages/osx/autopkg [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- Run autopkg with specified pkg:
+
+`autopkg run {{recipe_identifier}} --pkg {{/path/to/downloaded/pkg_or_dmg}}`
+
+- Update one or more recipe repo:
+
+`autopkg repo-update {{repo_names}}`
+
+`autopkg repo-update all`
```
Comment (fuerbringer) [inline pages/osx/autopkg]: Why not give this one a description aswell? Something like this:
```
- Update all repos:

`autopkg repo-update all`
```

This is again something that would've been picked up by `tldr-lint`.

File: pages/osx/autopkg (hunk 1 above)
Comment (Nerthil) [reply pages/osx/autopkg]: fixed, thx

File: pages/osx/autopkg.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# autopkg
+
+> Automation framework for macOS software packaging and distribution.
+
+- Run autopkg recipe:
+
+`autopkg run {{recipe_identifier}}`
+
+- Run autopkg with in verbose mode:
```
Comment (pxgamer) [inline pages/osx/autopkg.md]: Perhaps change this to `Run autopkg in verbose mode:` (just a minor grammatical change).

File: pages/osx/autopkg.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+`autopkg run -vvv {{recipe_identifier}}`
+
+- Run autopkg with specified pkg:
+
+`autopkg run {{recipe_identifier}} --pkg {{/path/to/downloaded/pkg_or_dmg}}`
```
Comment (pxgamer) [inline pages/osx/autopkg.md]: Could you remove the prefix `/` from the path?

**Comment by pxgamer** [comment]
This seems ok to me. :+1:

File: pages/osx/autopkg.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+`autopkg run {{recipe_identifier}}`
+
+- Run autopkg in verbose mode:
+
+`autopkg run -vvv {{recipe_identifier}}`
+
+- Run autopkg with specified pkg:
+
+`autopkg run {{recipe_identifier}} --pkg {{path/to/downloaded/pkg_or_dmg}}`
```
Comment (agnivade) [inline pages/osx/autopkg.md]: path/to/downloaded/pkg_or_dmg => path/to/downloaded_pkg_or_dmg

File: pages/osx/autopkg.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+# autopkg
+
+> Automation framework for macOS software packaging and distribution.
+
+- Run autopkg recipe:
```
Comment (agnivade) [inline pages/osx/autopkg.md]: We should add the `list-recipes` command first so that users can get the list of recipe identifiers to use in the subsequent commands.

**Comment by agnivade** [comment]
@Nerthil - The page has 9 examples. Our max limit has 8. I would recommend leaving out the verbose mode example. Or you can choose whichever one is the least important in the page. Once we do that, it should be good to go.

**Comment by agnivade** [comment]
Thanks @Nerthil. I made some minor grammatical changes. Please check and let me know whether it looks good to you.

File: pages/osx/autopkg.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+`autopkg run {{recipe_identifier}}`
+
+- Run autopkg with specified pkg:
+
+`autopkg run {{recipe_identifier}} --pkg {{path/to/downloaded_pkg_or_dmg}}`
+
+- Add a new repo to autopkg:
```
Comment (sbrl) [inline pages/osx/autopkg.md]: If I can add a new repo, do I need to install a recipe before using it? I'm not a macOS user, so I can't tell :P

Other than that question, looks ok to me! Thanks for the page, @Nerthil - and for being so patient with the review process 😺

Comment (Nerthil) [reply pages/osx/autopkg.md]: no, you don't have to use any repos for run a recipe but it's useful. 😄
